src/runtime/HalideBuffer.h

```diff
@@ -576,6 +576,13 @@ class Buffer {
         return (size_t)((const uint8_t *)end() - (const uint8_t *)begin());
     }
 
+    /** Reset the Buffer to be equivalent to a default-constructed Buffer
+     * of the same static type (if any); Buffer<void> will have its runtime
+     * type reset to uint8. The resulting buffer will have 0 dimensions and 1 element. */
```

Comment:
It won't really have one element, because it won't have an allocation. I'd just say it's equivalent to a default-constructed buffer of the same type.

Reply:
number_of_elements() == 1, but yes